xbmc/guilib/GUIIncludes.cpp

```diff
@@ -146,8 +146,21 @@ bool CGUIIncludes::LoadIncludesFromXML(const TiXmlElement *root)
         m_includes.insert({ tagName, { *includeBody, std::move(defaultParams) } });
     }
     else if (node->Attribute("file"))
-    { // load this file in as well
-      LoadIncludes(g_SkinInfo->GetSkinPath(node->Attribute("file")));
+    { 
+      const char *condition = node->Attribute("condition");
+      if (condition)
+      { // check this condition
+        INFO::InfoPtr conditionID = g_infoManager.Register(condition);
```

Comment:
thanx for reviewing!
i just c/p'd the code we use for conditionally including a piece of code:
https://github.com/ronie/xbmc/blob/1f41b7878ff16e7a3b7908a695c475c0d83a90ac/xbmc/guilib/GUIIncludes.cpp#L270